README.md

```diff
@@ -51,6 +52,21 @@ After you successfully set up the project, you will have to go through the follo
    * use it as the return value of the method `getBotToken()` inside `StickerifyBot`
 3. Run the `Main` class to start the bot, it will be now able to answer messages in Telegram
 
+
+## Launch The Bot Using Docker
+Prerequisites
+
+Make sure you have already installed both Docker Engine and Docker Compose. You don’t need to install other programs like Python  as they are provided by Docker images.
+
+1. You can see how to start with docker compose here.
+2. To run the bot locally  with Docker, we need to set Passing Environment Variables Into the Dockerfile you created in 1.
+3. To do that, When we want to set this argument, we pass it with the –build-arg flag:
```

Comment:
We can remove point 1 and 2, keeping this as the second one:
```suggestion
2. Run Docker passing it the argument variable with the following command:
```

Reply:
thank you for your suggestion, I have applied your suggestion.